utils/docker/Dockerfile-rpmbuild.sles.12.3

```diff
@@ -112,7 +112,8 @@ RUN zypper --non-interactive rm libpmem1;                                      \
                                      libpmemobj-devel raft-devel               \
                                      argobots-devel cart-devel                 \
                                      ipmctl-devel readline-devel               \
-                                     go1.10 fio\ \<\ 3.4
+                                     go1.10 fio\ \<\ 3.4;                      \
+    zypper --non-interactive rm go1.7 go1.8 go1.9
```

Comment:
Maybe leave a comment here explaining why this is necessary and what to do if the version of go is incremented? Go version 1.12 was released in February and 1.13 is due out late summer/early fall. It'd be ideal to figure out why this is necessary at all, of course, but at a minimum we should make sure that whomever updates this file knows how to prevent it from happening again.

Reply:
Agreed on figuring out why, but I don't know what our SUSE support options are and really don't have the time to dedicate to getting to that ultimate root cause.  Too many other things are demanding attention.

It could also just be that SLES 12.3 is going to be too old to even be supported, but it's the SLES version that is being dictated to us to support.

Reply:
SLES-12.3 support is only temporary.  We were requested to provide it as a special short term case.
Any extensive debugging will be under SLES 15/Leap 15 in the future.